### PR TITLE
Merge motion.update_metadata into motion.update

### PR DESCRIPTION
**Pull request by reiterl**
Merge the two action motion.update and motion.update_metadata into motion.update.
Also merge the tests.

### CODE REVIEW

**Review by jsangmeister — APPROVED** [review]
Much better this way ;)

**Comment by jsangmeister** [comment]
@GabrielInTheWorld @FinnStutzenstein This PR needs to be reflected in the client.

File: openslides_backend/action/actions/motion/update.py [hunk 1]
```diff
@@ -1,13 +1,23 @@
...
+    POSITIVE_NUMBER_REGEX,
+    Collection,
+    FullQualifiedId,
+)
+from ....shared.schema import optional_id_schema
 from ...generics.update import UpdateAction
 from ...util.default_schema import DefaultSchema
 from ...util.register import register_action
 
+RERIDS_PATTERN = re.compile(r"\[(?P<fqid>\w+/\d+)\]")
```
Comment (FinnStutzenstein) [inline openslides_backend/action/actions/motion/update.py:19]: what is `RERIDS`? Please find a better name

**Review by FinnStutzenstein — APPROVED** [review]

**Comment by FinnStutzenstein** [comment]
Tested together with https://github.com/OpenSlides/openslides-client/pull/130: Ok!

File: openslides_backend/action/actions/motion/update.py (hunk 1 above)
Comment (reiterl) [reply openslides_backend/action/actions/motion/update.py:19]: Replaced RERIDS_PATTERN with RECOMMENDATION_EXTENSION_REFERENCE_IDS_PATTERN.

**Pull request merged**